### ISSUE TITLE
Could you provide tokenized continue-pretraining dataset for reproduction?

### ISSUE THREAD

**Issue by gywlssww** [princeton-nlp/LLM-Shearing#51]
Could you provide tokenized continue-pretraining dataset for reproduction like pruning dataset?
Is tokenizer.model you provided exactly the same tokenizer as Llama-2?

**Comment by xiamengzhou** [comment]
Yes, we use the same tokenizer as llama-2. We'd love to share the data, but due to the shear amount of it, I am not sure what is the best way to serve it. Let me know if you have any idea!

**Comment by vmasrani** [comment]
+1! Would be very helpful to have the finetuning/continue-pretraining dataset as well to be able to reproduce paper results.
